### ISSUE TITLE
Duolingo has changed the colors about a couple of day ago

### ISSUE THREAD

**Issue by ghost** [derat/duoleavemealone#25]
Hi! Thank you for the awesome addon!
But Duolingo has changed the colors about a couple of day ago
and _duoleavemealone_ does not work on /learn lessons anymore
(on /stories it continues to work as designed). 
Please, repair it. 
Or is there a way for me to change colors codes by myself?

**Comment by derat** [comment]
Thanks for the heads-up. I'll hopefully have time to take a look tomorrow, but if you want to try to fix it before then, the `greenButtonColor`, `correctDivColor`, etc. constants starting on L129 of `content.js` are probably what need to change. That's assuming that they just changed the colors and didn't make any other changes to the structure of the page, though.

**Comment by derat** [comment]
Haven't tested it yet, but from running through a lesson, the colors might just need to change to this:

```js
const greenButtonColor = '#58cc02';
const correctDivColor = '#d7ffb8';
const correctMessageColor = '#58a700';
```

**Comment by derat** [comment]
I've submitted version 0.21 of the extension for review, so hopefully the fix will go out soon. It looks like they only changed the background color of the "correct" message -- at least, updating that was enough to get the extension working again on a practice skill. Let me know if anything else is still broken after you get the update.

**Comment by ghost** [comment]
Unfortunately, now they randomize the color of Continue button on each new question.

**Comment by derat** [comment]
Hmm, interesting -- thanks for pointing that out. It looks to me like the continue button's background color is `#58a700` after the first question and then `#58cc02` after later questions. Is that what you're seeing too?

I'm guessing this is just a bug on their side. I'll update the extension to look for either color.

**Comment by derat** [comment]
Sigh, I can't upload 0.22 to the Chrome Web Store since 0.21 is still under review.

**Comment by duoleague** [comment]
There is no need to update Chrome Web Stor for each try, I use developer mode in my chrome.
Yesterday I saw a new color code for each question, today I saw some dark green color on first answer and now I see rgb(88, 204, 2) on each Continue button and use this color in add-on, but it still does not work for me.
Well, thank you. I am sick and tired of tricks and traps of Duolingo, that's enough for me, I'm leaving the site.
All their time they think and care about coding games, not about studing languages.
